src/styles/ia-bookmarks-list.js

```diff
@@ -0,0 +1,57 @@
+import { css } from 'lit-element';
+
+export default css`
+:host {
+  display: block;
+  padding: 0 1rem;
+  color: var(--primaryTextColor);
+}
+
+small {
+  font-style: italic;
+}
+
+ul {
+  padding: 0;
+  margin: 0;
+  list-style: none;
+}
+
+li {
+  display: grid;
+  grid-template-columns: 37px 1fr auto;
+  grid-gap: .2rem 1rem;
+  justify-content: space-between;
+  align-items: center;
+  cursor: pointer;
+  border: 5px solid transparent;
+}
+
+.active {
+  border-color: var(--activeBookmark);
+}
+
+h4 {
+  margin: 0;
+  font-size: 1.4rem;
+}
+
+p {
+  padding: 0;
+  margin: 0;
+  grid-column: 1 / 4;
+}
+
+button {
+  -webkit-appearance: none;
+  appearance: none;
+  width: 20px;
```

Comment:
please use rem where you can

Reply:
This is the width for the button containing the SVG icon, whose size will remain unchanged as the relative font size changes.